resources/deprecations.properties

```diff
@@ -152,9 +167,11 @@ setenv = https://wiki.jenkins.io/x/YAtSAg
 sonargraph-plugin = https://git.io/JJ5h1
 # https://wiki.jenkins.io/pages/viewpage.action?pageId=63930505
 sonatype-ci = https://wiki.jenkins.io/x/iYDPAw
+svn-tag = https://jenkins.io/security/advisory/2017-04-10/
 swamp = https://issues.jenkins.io/browse/INFRA-2487
 tasks = https://issues.jenkins.io/browse/INFRA-2487
 # https://wiki.jenkins.io/display/JENKINS/TEPCO+Plugin
+tfs = https://issues.jenkins.io/browse/INFRA-2751
```

Comment:
is it not better to link to the [advisory](https://www.jenkins.io/security/advisory/2021-03-30/)? the issue... well the reason for suspension could easily be lost in history if someone else comments.

Reply:
There are two advisories for TFS.  The INFRA ticket can be updated and contains more history.  I think we should continue with INFRA

Reply:
The security issues are not the reason for its suspension, so that would be wrong.

Reply:
I thought the licensing issue was resolved? So what’s the reason still? Just no maintainer?

Reply:
Nobody stepping up to maintain it 🤷